package.json

```diff
@@ -8,34 +8,41 @@
     "node": "22.17.0"
   },
   "scripts": {
-    "ci": "npm install && npm run generate && npm run package && npm run type && (npm run lint:ci || true) && npm run test:ci",
-    "generate": "npm run -w @axonivy/ui-icons generate",
+    "preinstall": "npx only-allow pnpm",
+    "ci": "pnpm install && pnpm run generate && pnpm run package && pnpm run type && (pnpm run lint:ci || true) && pnpm run test:ci",
+    "generate": "pnpm run -r generate",
     "build": "tsc --build",
     "build:storybook": "storybook build",
-    "package": "lerna run package",
-    "clean": "lerna run clean",
-    "type": "lerna run type",
+    "package": "pnpm run -r package",
+    "clean": "pnpm run -r clean",
+    "type": "pnpm run -r type",
     "lint": "eslint",
     "lint:ci": "eslint -o eslint.xml -f checkstyle",
     "lint:fix": "eslint --fix",
     "lint:inspect": "eslint --inspect-config",
     "dev": "storybook dev -p 6006 --no-open",
     "test": "vitest",
     "test:ci": "vitest run --reporter=default --reporter=junit --outputFile=report.xml",
+    "raise:version": "lerna version --allow-peer-dependencies-update --no-git-tag-version --no-push --ignore-scripts --exact --yes",
     "publish:next": "lerna publish --exact --canary --preid next --pre-dist-tag next --no-git-tag-version --no-push --ignore-scripts --yes"
   },
   "devDependencies": {
+    "@axonivy/eslint-config": "workspace:*",
```

Comment:
why is this needed?

Reply:
Because otherwise the root does not know the eslint bin and cannot trigger the linting

Reply:
ah sorry lint is obvious, but why is `"@axonivy/ui-components": "workspace:*",` declared here?

Reply:
because storybook is in the root and it needs the styles from it